apps/condo/domains/miniapp/schema/MessageAppBlackList.js

```diff
@@ -10,7 +10,7 @@ const { getMessageTypeField } = require('@condo/domains/notification/schema/fiel
 
 
 const MessageAppBlackList = new GQLListSchema('MessageAppBlackList', {
-    schemaDoc: 'Rule for blocking specific type of messages for apps',
+    schemaDoc: '[Deprecated] Rule for blocking specific type of messages for apps',
```

Comment:
Does it mean, that in the future that model will be completely removed?

Reply:
yes